electron/legendary/games.ts

```diff
@@ -724,6 +732,23 @@ class LegendaryGame extends Game {
       return logInfo(`${pattern} killed`, LogPrefix.Legendary)
     })
   }
+
+  public forceUninstall(appName: string, runner: string) {
+    // Modify Legendary installed.json file:
```

Comment:
Why not just run `legendary uninstall AppName -y --keep-files`?

Also, since this is in the LegendaryGame class, can't we use `this.appName` for the AppName and just `'Legendary'` as the runner?

Reply:
Yes, didnt know about this command. I will try.

Reply:
works like charm :)